test/karma.conf.js

```diff
@@ -111,10 +111,10 @@ module.exports = function(config) {
       },
       bs_ios_15: {
```

Comment:
```suggestion
      bs_ios_14: {
```
(why did this need to be changed btw?)

Reply:
Safari 15 does not expose WebCrypto on localhost 😢 #1624